### ISSUE TITLE
Tagged PDF support

### ISSUE THREAD

**Issue by mwcampbell** [QuestPDF/QuestPDF#193]
Unless I'm missing something, it appears that this library doesn't support generation of tagged PDF documents. This feature is important for accessibility. Is this already on the project roadmap? Note: I'm not a user or potential user of this library; I just came across it and felt that this missing functionality was an important thing to point out. I appreciate that this library is freely available under a permissive license, but I would still be disappointed if its adoption leads to inaccessible documents.

**Comment by MarcinZiabek** [comment]
Hello 😁 QuestPDF acts as advanced document layout engine and uses SkiaSharp to produce final PDF file. Therefore, it derives all SkiaSharp limitations. At this point, as far as I know, SkiaSharp does not support tagget PDF files. 

However, having bigger community and financial resources, it is viable to investigate writing custom PDF renderer with tags support. My intent is to provide more and more features over time, including accessability improvements.

**Comment by MarcinZiabek** [comment]
I’m sorry to confirm that tagged PDF documents are not currently supported. However, thanks to recent enhancements, implementing this feature is now feasible. I’ve added it to the backlog and will evaluate its priority on the roadmap.

I’m closing this issue due to inactivity. Thank you for your feedback! ❤️
